axelar/infrastructure/asg.tf

```diff
@@ -242,3 +100,145 @@ resource "aws_instance" "tmkms" {
     Environment = "testnet"
   }
 }
+
+resource "aws_instance" "ethereum_node" {
+  ami           = "ami-011899242bb902164"
+  instance_type = "m5.xlarge"
+
+  ebs_optimized = true
+  root_block_device {
+    delete_on_termination = false
+
+    volume_type = "gp3"
+    volume_size = 1000
+  }
+
+  key_name = aws_key_pair.ssh.id
+  private_ip = terraform.workspace == "axelar-testnet" ? "10.0.11.31" : "10.0.11.8"
```

Comment:
what workspace will be expected here? not `test` || `prod`?

Reply:
no i did it based on the aws profile to make it easier
so workspace names are actually axelar-testnet, and axelar-mainnet